code/src/cljs/sixsq/nuvla/ui/edges/events.cljs

```diff
@@ -61,6 +68,9 @@
     {:fx [[:dispatch [::main-events/action-interval-start {:id        refresh-id
                                                            :frequency 10000
                                                            :event     [::get-nuvlaboxes]}]]
+          [:dispatch [::main-events/action-interval-start {:id        refresh-id-non-edit
```

Comment:
is this recurrent search needed? I see a load of them on open of bulk edit modal

Reply:
I wanted to use it outside the modal too to show the number in "Select all XX". But I changed that back again.

So yes: not needed anymore.